content/en/tracing/runtime_metrics/dotnet.md

```diff
@@ -0,0 +1,50 @@
+---
+title: .NET Runtime Metrics
+kind: documentation
+description: "Gain additional insights into your .NET application's performance with the runtime metrics associated to your traces."
+further_reading:
+    - link: 'tracing/connect_logs_and_traces'
+      tag: 'Documentation'
+      text: 'Connect your Logs and Traces together'
+    - link: 'tracing/manual_instrumentation'
+      tag: 'Documentation'
+      text: 'Manually instrument your application to create traces.'
+    - link: 'tracing/opentracing'
+      tag: 'Documentation'
+      text: 'Implement Opentracing across your applications.'
+    - link: 'tracing/visualization/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+## Automatic Configuration
+
+Enable runtime metrics collection in the .NET Tracer 1.22.0+ with the `DD_RUNTIME_METRICS_ENABLED=true` environment variable.
+
+View runtime metrics in correlation with your .NET services. See the [Service page][1] in Datadog.
```

Comment:
To maintain consistent wording as other language pages:

```suggestion
Runtime metrics can be viewed in correlation with your .NET services. See the [Service page][1] in Datadog.
```

Reply:
I just realized that this was recommended by @apigirl . The text `View runtime metrics in correlation with your .NET services. See the [Service page][1] in Datadog.` aligns with the text for the other languages. Which text should we use across languages? I have opened a new PR to add more consistency across the language's runtime metrics pages: #9492

Reply:
That PR has been merged already, but I can open a new PR for standardizing text across the languages